2019_15_R_for_ArcGISusers.Rproj

```diff
@@ -13,5 +13,3 @@ RnwWeave: Sweave
 LaTeX: pdfLaTeX
 
 BuildType: Website
-
-SpellingDictionary: en_GB
```

Comment:
you are still committing various files

Reply:
I know. Although I made the commit via gui, the push was not not possible